tutor/src/components/icons/extension.js

```diff
@@ -0,0 +1,27 @@
+import { React, styled } from 'vendor';
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
+
+const GreenCircle = styled.div`
+  display: inline-block;
+  width: 1.4rem;
+  height: 1.4rem;
+  position: absolute;
+  right: 1rem;
+  background: #009670;
+  color: #fff;
+  font-size: 0.9rem;
+  line-height: 1.4rem;
+  border-radius: 50%;
+  text-align: center;
+`;
+
```

Comment:
this seems like it would be useful on the other screens?  If so how about extracting the: `<GreenCircle>E</GreenCircle>` and put it in `components/icons/extension.js` ?

Reply:
This also made me realize I need to move the position stuff to be specific to the scores table, so thanks :D